pkg/client/cache/listers.go

```diff
@@ -123,6 +123,61 @@ func (s *StoreToNodeLister) GetNodeInfo(id string) (*api.Node, error) {
 	return minion.(*api.Node), nil
 }
 
+// StoreToControllerLister gives a store List and Exists methods. The store must contain only ReplicationControllers.
+// Note that this lister matches the given selectors against the rcs Spec.Selectors, not its Spec.Labels.
```

Comment:
s/rcs/RC's/

Recommend renaming the List methods ListBySelector to make it clear that they are not like ordinary lists.